.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,69 @@
+name: Deploy alerts to all environments
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - .github/workflows/deploy.yaml
+      - .nais/meldeplikt-alerts.yaml
+      - .nais/vars.yaml
+      - .nais/alerts-common.yaml
+      - .nais/alerts-fss.yaml
+      - .nais/alerts-gcp.yaml
+  workflow_dispatch:
+
+env:
+  RESOURCE_FILE: .nais/meldeplikt-alerts.yaml
+  RESOURCE_FILE_FSS: .nais/meldeplikt-alerts-prod-fss.yaml
+  RESOURCE_FILE_GCP: .nais/meldeplikt-alerts-prod-gcp.yaml
+  VARS_FILE: .nais/vars.yaml
+
+jobs:
+  deploy_dev:
+    name: Deploy alerts to all environments
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Append alerts
+        run: |
+          cat .nais/alerts-common.yaml >> ${{ env.RESOURCE_FILE }}
+          cat ${{ env.RESOURCE_FILE }} .nais/alerts-fss.yaml > ${{ env.RESOURCE_FILE_FSS }}
+          cat ${{ env.RESOURCE_FILE }} .nais/alerts-gcp.yaml > ${{ env.RESOURCE_FILE_GCP }}
+
+      - name: Deploy to dev-fss
+        uses: nais/deploy/actions/deploy@v1
+        env:
+          APIKEY: ${{ secrets.NAIS_DEPLOY_APIKEY }}
+          CLUSTER: dev-fss
+          RESOURCE: ${{ env.RESOURCE_FILE_FSS }}
```

Comment:
Vi deployer til dev her, men bruker env.RESOURCE_FILE_FSS som er .nais/meldeplikt-alerts-prod-fss.yaml

Reply:
Godt poeng, endrer filnavn til `.nais/meldeplikt-alerts-fss.yaml`

Reply:
Men må vi ikke har dev fil her? Vi jo deployer til dev-fss her

Reply:
Nei, innholdet i filen er det samme for prod og dev. Men det ble litt misvisende å ha "prod" i navnet når den brukes både ved deploy til prod og dev.